### PR TITLE
WaterTask documentation

### PR DESCRIPTION
**Pull request by nikhil-2101**
Documentation for WaterTask

### CODE REVIEW

**Comment by SuveenE** [comment]
LGTM!

File: src/main/java/activeedge/task/WaterTask.java [hunk 1]
```diff
@@ -2,23 +2,47 @@
 
 import java.util.ArrayList;
 
+/**
```
Comment (Praneet-25) [inline src/main/java/activeedge/task/WaterTask.java:5]: Documentation is detailed and explains the functions properly.

File: src/main/java/activeedge/task/WaterTask.java [hunk 2]
```diff
@@ -2,23 +2,47 @@
...
+     * @return A string representing the water task and its quantity.
+     */
     @Override
     public String toString() {
         return "Water " + this.getQuantity();
     }
 
+    /**
+     * Calculates the total water intake from a list of tasks.
+     *
```
Comment (Praneet-25) [inline src/main/java/activeedge/task/WaterTask.java:42]: Clear explanation

**Review by Praneet-25 — COMMENTED** [review]
LGTM!

**Pull request merged**